docs/intellicode-visual-studio-code.md

```diff
@@ -16,24 +16,43 @@ The [IntelliCode extension for Visual Studio Code](https://marketplace.visualstu
 
 ![IntelliCode for Python in Visual Studio Code](media/python-intellicode.gif)
 
-## Supported languages
-
 The supported languages for AI-assisted IntelliSense completions are:
 
 - Java
 - JavaScript and TypeScript
 - Python
 - SQL
 
-## Prerequisites
 
-To install the extension, you need Visual Studio Code October 2018 Release 1.29 or later.
+### Prerequisites
+
+The extension is supported by Visual Studio Code October 2018 Release 1.29 or later.
+
+AI-assisted IntelliSense for Java requires Java 8 Update 151 or higher. 
+## Whole-line autocomplete preview
+
+The [IntelliCode Completions extension for Visual Studio Code](https://marketplace.visualstudio.com/items?itemName=VisualStudioExptTeam.vscodeintellicode-completions) is an experimental extension which predicts up to a whole line of code based on your current context. Predictions appear as grey-text to the right of your cursor. This extension supports Python, with additional experimental support for JavaScript and TypeScript.
+
+![IntelliCode Completions for Python in Visual Studio Code](../images/wlc.gif)
+
+Completions will appear after your cursor as you type, with a faded color. At any time, you can accept the suggestion by pressing the tab key. Additionally, you can dismiss any shown suggestion by pressing the ESC key.
+
+![IntelliCode Completions interpreting IntelliSense completion list selection in Visual Studio Code](../images/intellisenseSelection.gif)
 
-To get AI-assisted IntelliSense for Java, you need Java 8 Update 151 or higher. (If you have a previous Java 8 version, you'll see the error **java.security.InvalidKeyException: Illegal key size**, and IntelliCode AI-assisted IntelliSense recommendations won't work.)
+To enable experimental support for JavaScript and TypeScript, search for intellicodeCompletions.language in the Settings menu.
```

Comment:
changing the font of `intellicodeCompletions.language` this by either highlighting it or using ` would make it easier to read